eth/stagedsync/stage_headers.go

```diff
@@ -57,6 +57,50 @@ func StageHeadersCfg(
 	}
 }
 
+func SpawnStageHeaders(
```

Comment:
`SpawnStageHeaders` isn't used anywhere, is it?

Reply:
I now call it, however it does nothing, it is just a starting point for downward download

Reply:
Hmm. It looks like using `SpawnStageHeaders` in https://github.com/ledgerwatch/erigon/pull/2997/commits/19785b42c46c19318fe696ecbedbf523a596af6a is causing the tests to fail.

I'd suggest to remove `SpawnStageHeaders` for now to reduce the scope of this PR.

Reply:
okay removed it for now